### PR TITLE
About Me feature complete

### PR DESCRIPTION
**Pull request by srf6413**
Everything should be all set. I did notice that before and after I sanitized user input , when I entered "<script>alert("Pay me a million dollars or I'll delete your files!");</script>" into the about me and posted, it did nothing instead of giving a javascript message. Let me know if there is something I missed but I'm pretty sure I followed the instructions exactly as they were...

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -100,4 +100,35 @@ public void storeMessage(Message message) {
     }
     return messages;
   }
+  /** 
+   *Stores the User in Datastore.
+   */
+  
+  public void storeUser(User user) {
+
```
Comment (ctrokid) [inline src/main/java/com/google/codeu/data/Datastore.java]: Nitpicky: could we remove this blank line?

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -100,4 +100,35 @@ public void storeMessage(Message message) {
...
+    userEntity.setProperty("aboutMe", user.getAboutMe());
+    datastore.put(userEntity);
+  }
+
+  /**
+   * Returns the User owned by the email address, or
+   * null if no matching User was found.
+   */
+  public User getUser(String email) {
+
```
Comment (ctrokid) [inline src/main/java/com/google/codeu/data/Datastore.java]: Same thing with this blank line?

**Review by ctrokid — CHANGES_REQUESTED** [review]
Nice, this looks good! I think not seeing the alert show up means that you are correctly sanitizing the user input.

Only question I have is if you were successfully able to deploy this to App Engine yet?

After that question and those two small fixes, I will approve.

**Comment by srf6413** [comment]
When I took the whitespace out, now travis is giving me an error:(

**Review by ctrokid — APPROVED** [review]
This LGTM. You'll just have to figure out the lint errors before it will let you submit 😛

**Pull request merged**